renew.py

```
@@ -5,16 +5,16 @@
 PORT = int(os.environ.get("PORT", 5000))
 
 login_data = {
-    "username": "zatarainbown", # đổi thành username acc mcserver vào đây
+    "username": "quangten", # đổi thành username acc mcserver vào đây
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)